### ISSUE TITLE
Not able to select a region other than the default for an Imported GKE v2 cluster

### ISSUE THREAD

**Issue by sowmyav27** [rancher/rancher#32044]
**What kind of request is this (question/bug/enhancement/feature request):** bug

**Steps to reproduce (least amount of steps as possible):**

- Create an imported GKE v2 cluster.
- Enter cloud creds and the project ID.
- Select region - us-central1 or any region other than the default `us-west2`
- The Request URL is `https://<rancher-server>/meta/gkeClusters?cloudCredentialId=cattle-global-data%3Acc-8sb64&region=us-west2&projectId=<project-id>`
- Region does not change

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): 2.5-head commit id: `d1083bc63`
- Installation option (single install/HA): single

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): Imported GKE v2

**Comment by aaronRancher** [comment]
This is also seen when selecting Zones. On `rancher\rancher:master-head` `version b202f20` and `rancher\rancher:v2.5-head` `version d1083bc`.

**Comment by thehejik** [comment]
It's fixed in `rancher:v2.5-head` `7002499` and in `rancher:master-head` `6012944` as well

I was trying to set different regions and it was always sending a new request with corresponding `region` in it:
```
https://<my-rancher>/meta/gkeClusters?cloudCredentialId=cattle-global-data%3Acc-ck6mt&region=us-central1&projectId=rancher-qa
https://<my-rancher>/meta/gkeClusters?cloudCredentialId=cattle-global-data%3Acc-ck6mt&region=europe-north1&projectId=rancher-qa
https://<my-rancher>/meta/gkeClusters?cloudCredentialId=cattle-global-data%3Acc-ck6mt&region=southamerica-east1&projectId=rancher-qa
```

Closing.
